lib/initial-setup/WP_Auth0_InitialSetup_Consent.php

```diff
@@ -128,19 +144,21 @@ public function consent_callback( $name ) {
 		$connection_exists     = false;
 		$connection_pwd_policy = null;
 
-		$connections = WP_Auth0_Api_Client::search_connection( $domain, $app_token );
+		$connections = WP_Auth0_Api_Client::search_connection( $domain, $app_token, 'auth0' );
```

Comment:
Added strategy filter here to reduce the number of Connections returned.

Reply:
Public static method so no, not currently. `WP_Auth0_Api_Client` is getting refactored one method at a time so it will go away soon.